gravity/proto/gravity.proto

```
@@ -0,0 +1,555 @@
+syntax = "proto3";
+
+package gravity;
```

Buf (1.55.1)

Files with package "gravity" must be within a directory "gravity" relative to root but were in directory "gravity/proto".

(PACKAGE_DIRECTORY_MATCH)